.vscode/tasks.json

```
@@ -0,0 +1,65 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Install dependencies",
+            "type": "shell",
+            "command": "pip install debugpy \"sqlalchemy~=1.3.22\" \"geoalchemy2>=0.9.0\"",
+            "windows": {
+                "options": {
+                    "shell": {
+                        "executable": "C:\\OSGeo4W\\OSGeo4W.bat"
+                    }
+                }
+            },
+            "problemMatcher": []
+        },
+        {
+            "label": "docker-db",
+            "type": "docker-compose",
+            "dockerCompose": {
+                "up": {
+                    "detached": true,
+                    "build": false,
+                    "services": [
+                        "db"
+                    ]
+                },
+                "files": [
+                    "${workspaceFolder}/docker-compose.yml"
+                ]
+            }
+        },
+        {
+            "label": "Launch QGIS",
+            "type": "process",
+            "args": [
+                "--project",
+                "${workspaceFolder}\\project\\teksi_wastewater.qgs"
+            ],
+            "runOptions": {
+                "instanceLimit": 1,
+                "reevaluateOnRerun": false
+            },
+            "dependsOn": [
+                "docker-db"
+            ],
+            "problemMatcher": [],
+            "options": {
+                "env": {
+                    "QGIS_PLUGINPATH": "${workspaceFolder}\\plugin",
+                    "QGIS_PLUGIN_USE_DEBUGGER": "debugpy",
+                    "PGSERVICEFILE": "${workspaceFolder}\\project\\pg_service.conf"
+                }
+            },
+            "windows": {
+                "command": "C:\\OSGeo4W\\bin\\qgis-ltr-bin.exe"
+            },
+            "linux": {
+                "command": "qgis-ltr"
```

```suggestion
                "command": "qgis"
```

I guess qgis-ltr is not available by default on most distros

---

I understood that the development and testing should be conducted against latest ltr release. Does the ltr release lack the `qgis-ltr` binding and is only available as `qgis`? Can multiple versions even coexist on linux like they are in OSGeo4w?

---

> I understood that the development and testing should be conducted against latest ltr release.

Right but in fact if you install the ltr, or another release it is always installed as `/usr/bin/qgis` (at least in the distro I am using). 
In practice I usually work with a slighter newer version for convenience. Note that the pipeline tests are run against 3.34 and latest https://github.com/teksi/wastewater/blob/main/.github/workflows/plugin-test.yml#L32

> Does the ltr release lack the `qgis-ltr` binding and is only available as `qgis`? Can multiple versions even coexist on linux like they are in OSGeo4w?

It depends how it is packaged, but to get a specific qgis version you usually need to add additional repositories.
For example in OpenSUSE you can add the Geo repo and you will get the option to install the ltr but this way you have to choose ltr or latest.

To have different versions at the same time I usually build them from source and install into a custom prefix.